solidity/dashboard/src/css/typography.less

```diff
@@ -140,28 +159,9 @@ label {
     font-weight: normal !important;
   }
 
-  &-label {
-    font-size: 0.8125rem;
-    color: @color-grey-60;
-    line-height: 15px;
-    letter-spacing: 0.08em;
-    text-transform: uppercase;
-    font-feature-settings: 'tnum' on, 'lnum' on;
-  }
-
-  &-big {
-    font-weight: normal;
-    font-size: 1.125rem !important;
-    line-height: 145%;
-    color: @color-grey-60;
-    font-feature-settings: 'tnum' on, 'lnum' on;
-  }
-
   &-samll {
```

Comment:
Looks like a misspelling that's probably percolated through everything :grimacing:

Reply:
heh 🤦  in UI components we use `text-small` 😄. It's probably the only place with this misspelling, I'll update.

Reply:
pls see 092322f